html/index.html

```diff
@@ -110,6 +115,11 @@ <h4>Navigation</h4>
         </div>
     </header>
     <main>
+        <div>
+            <button id="main-call-button">Refresh Images</button>
+            <div id="success-alert-placeholder"></div> 
```

Comment:
use bootstrap code for this:
https://getbootstrap.com/docs/5.0/components/alerts/

Reply:
plz make it more universal